gui.py

```diff
@@ -526,6 +533,14 @@ def read_ui_settings(self):
 
         self.settings.beginGroup('treeViewMix')
         self.treeViewMix.read_settings(self.settings, 'ColumnWidth')
+
```

Comment:
Please don't comment out things, just remove them. We need to reduce visual noise in our codebase. Thanks